Makefile

```diff
@@ -242,8 +241,8 @@ catalog-retag-latest:
 ##@ Kind Deployment
 
 kind-create: export KUBECONFIG = $(PWD)/kubeconfig
-kind-create: tmp docker-build kind ## Runs a k8s kind cluster with a local registry in "localhost:5000" and ports 1080 and 1443 exposed to the host
-	$(KIND) create cluster --wait 5m --image kindest/node:v1.21.0
+kind-create: docker-build kind ## Runs a k8s kind cluster with a local registry in "localhost:5000" and ports 1080 and 1443 exposed to the host
+	$(KIND) create cluster --wait 5m --image kindest/node:v1.20.7
```

Comment:
downgrade of kind? why?

Reply:
Because we are currently using k8s 1.20 in stg and pro, better to test with the same version also in kind

Reply:
fair enough!